_data/talks.yml

```diff
@@ -1,5 +1,21 @@
 # Data file that describes talks
 
+- name: 'Progressive Web Apps mit der Service Worker API'
+  author: 'simon'
+  summary: >
+    Progressive Web Apps (PWAs) schließen die Lücke zwischen Webseiten und nativen Applikationen. 
```

Comment:
```suggestion
Progressive Web Apps (PWAs) bridge the gap between websites and native applications. 
They combine the high accessibility of web pages with the open standards of the web in terms of 
Look and feel (HTML, CSS), search engine optimization (SEO) and functionality (Push API, Notification API), 
Web Storage API, HTTPS etc.) with some properties of native applications. So Progressive 
Web Apps offline (Service Worker API) and from the Home Screen (Web App Manifest).

This talk shows how to create a PWA. The focus is on the Service Worker API, 
which plays a central role for Progressive Web Apps. It will also show how the Web 
App manifest and which tools help to realize a PWA.
```